test/trivy_ignore_test.go

```diff
@@ -0,0 +1,393 @@
+package test
```

Comment:
Likewise with this file as well. `ignore_test.go` could include all the ignores, trivy and tfsec as test inputs for each of the existing tests within `ignore_test.go`.